modules/custom/openy_system/src/Form/OpenyModulesListForm.php

```diff
@@ -76,29 +76,35 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   /**
    * Builds a table row for the OpenY packages  page.
    *
-   * @param array $module
+   * @param array $package
    *   The package for which to build the form row.
-   * @param $distribution
    *
    * @return array
    *   The form row for the given module.
    */
-  protected function buildPackageRow(array $module) {
+  protected function buildPackageRow(array $package) {
     // Set the basic properties. Should be present to avoid notices in template_preprocess_system_modules_details()
     $row['#required'] = [];
     $row['#requires'] = [];
     $row['#required_by'] = [];
-    // Get human readable names of  modules in package.
+
+    $modules = system_rebuild_module_data();
+    // Get human readable names and status of modules in package.
     $module_names = [];
-    foreach ($module['modules'] as $name) {
-      $module_names[] = $this->moduleHandler->getName($name);
+    foreach ($package['modules'] as $name) {
+      if ($modules[$name]->status) {
+        $module_names[] = $modules[$name]->info["name"];
+      }
+      else {
+        $module_names[] = $modules[$name]->info["name"] . $this->t(' (disabled)');
```

Comment:
please do not add blank into the `t()` function. I would prefer to see the following implementation 
`$module_names[] = $modules[$name]->info["name"] . ' (' . $this->t('disabled') . ')'`